src/components/DeleteAccount/FormModal.jsx

```diff
@@ -17,91 +17,80 @@ const SENDING = 'sending'
 
 const REASON_MAXLENGTH = 3000
 
-export class FormModal extends Component {
-  state = {
-    status: IDLE
-  }
+const FormModal = ({ onSuccess, onError, onClose }) => {
+  const [status, setStatus] = useState(IDLE)
+  const client = useClient()
+  const { t } = useI18n()
+  const reasonElementRef = useRef()
 
-  setStatus = status => this.setState({ status })
+  const isSending = status === SENDING
 
-  onSuccess = () => {
-    const { onSuccess } = this.props
-    this.setStatus(DONE)
+  const handleSuccess = () => {
+    setStatus(DONE)
     onSuccess && onSuccess()
   }
 
-  onError = error => {
-    const { onError } = this.props
-    this.setStatus(ERRORED)
+  const handleError = error => {
+    setStatus(ERRORED)
     onError && onError(error)
   }
 
-  onSend = async event => {
+  const handleSend = async event => {
     event.preventDefault()
-    const { client, t } = this.props
     const STACK_DOMAIN = getStackDomain()
     const domain = STACK_DOMAIN.replace('//', '')
-    const reason = this.reasonElement.value
-    this.setStatus(SENDING)
+    const reason = reasonElementRef.current.value
+    setStatus(SENDING)
+
     try {
-      await sendDeleteAccountRequest(
+      await sendDeleteAccountMail(
```

Comment:
Why do we still call this, Shouldn't this be done by the cozy-stack now?

Reply:
il y a 2 mails à envoyer : un mail de confirmation de suppression (fait par la stack), un mail qui indique la raison de la suppression (fait ici). Méthode à renommer peut-être ?

Reply:
Ah ok, c'est un mail de feedback à envoyer à Cozy donc?

Du coup `sendReasonToCozy()` ? (limite on s'en fout de savoir si c'est par mail ou non à cet endroit)